src/generators/schema/howto.php

```diff
@@ -46,9 +46,9 @@ private function add_duration( &$data, $attributes ) {
 			return;
 		}
 
-		$days    = empty( $attributes['days'] ) ? 0 : $attributes['days'];
-		$hours   = empty( $attributes['hours'] ) ? 0 : $attributes['hours'];
-		$minutes = empty( $attributes['minutes'] ) ? 0 : $attributes['minutes'];
+		$days    = ! \is_numeric( $attributes['days'] ) ? 0 : \intval( $attributes['days'] );
+		$hours   = ! \is_numeric( $attributes['hours'] ) ? 0 : \intval( $attributes['hours'] );
+		$minutes = ! \is_numeric( $attributes['minutes'] ) ? 0 : \intval( $attributes['minutes'] );
```

Comment:
* You no longer check if the key in `$attributes` exists, which is what `empty` also did. Which will result in a PHP notice or warning (depending on PHP version).
* In general, negating in a ternary is strange, you can just switch the operation itself

Reply:
Incorporated feedback from above and Slack in a new version.